tests/foreman/ui/test_contenthost.py

```diff
@@ -262,23 +266,29 @@ def test_positive_end_to_end_bulk_update(session, default_location, vm):
             }
         )
         session.hostcollection.associate_host(hc_name, vm.hostname)
-        # make a note of time for later CLI wait_for_tasks, and include
-        # 8 mins margin of safety.
-        timestamp = (datetime.utcnow() - timedelta(minutes=8)).strftime('%Y-%m-%d %H:%M')
+        # For BZ#1838800, assert the Host Collection Errata Install table has the search URI
+        uri = session.hostcollection.search_applicable_hosts(hc_name, FAKE_1_ERRATA_ID)
+        assert search_uri in uri
+        # Ensure Errata dialog is closed
+        session.browser.click(
+            locator="(//div[contains(@class, 'modal-header')]/.//button[@ng-click='cancel()'])"
+        )
+        # Note time for later wait_for_tasks, and include 4 mins margin of safety.
+        timestamp = (datetime.utcnow() - timedelta(minutes=4)).strftime('%Y-%m-%d %H:%M')
         # Update the package by name
         session.hostcollection.manage_packages(
             hc_name,
-            content_type='Package',
+            content_type='rpm',
             packages=FAKE_1_CUSTOM_PACKAGE_NAME,
             action='update_all',
             action_via='via remote execution',
         )
         # Wait for upload profile event (in case Satellite system slow)
-        host = entities.Host().search(query={'search': f'name={vm.hostname}'})
         wait_for_tasks(
-            search_query='label = Actions::Katello::Host::UploadProfiles'
-            f' and resource_id = {host[0].id}'
-            f' and started_at >= "{timestamp}"',
+            search_query='label = Actions::Katello::Applicability::Hosts::BulkGenerate'
+            f' and started_at >= "{timestamp}"'
+            f' and state = stopped'
+            f' and result = success',
```

Comment:
Why its been moved from `host tasks to wait for` to `a collection of hosts` tasks, is this applies to more than one host ?

Reply:
Hi @jyejare 

`Actions::Katello::Host::UploadProfiles` does not exist in Sat7

see:
 Change applicability task check #9323